README.md

```diff
@@ -43,7 +56,7 @@ learning frameworks.
 - Distributed training support.
 - Configuration file examples for each supported compression algorithm.
 - Git patches for prominent third-party repositories ([huggingface-transformers](https://github.com/huggingface/transformers)) demonstrating the process of integrating NNCF into custom training pipelines
-- Exporting PyTorch compressed models to ONNX\* checkpoints and TensorFlow compressed models to SavedModel or Frozen Graph format, ready to use with [OpenVINO&trade; toolkit](https://github.com/openvinotoolkit/).
+- Exporting PyTorch compressed models to ONNX\* checkpoints and TensorFlow compressed models to SavedModel or Frozen Graph format, ready to use with [OpenVINO&trade; toolkit](https://docs.openvino.ai/latest/home.html).
 - Support for [Accuracy-Aware model training](./docs/Usage.md#accuracy-aware-model-training) pipelines via the [Adaptive Compression Level Training](./docs/accuracy_aware_model_training/AdaptiveCompressionLevelTraining.md) and [Early Exit Training](./docs/accuracy_aware_model_training/EarlyExitTraining.md).
 
 ## Usage
```

Comment:
The usage section is very huge and is not consistent and requires rethinking. I'm ok if it will be done in the  another PR, but if you have idea how to do it, I'm ready to discuss.

Reply:
I think that PTQ section looks good. There are minimalistic instructions aim to make an introduction of using PTQ. 
Could you specify the exact points that look overflowed?